FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Chris on 2021-05-27.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+import CoreData
+
+@objc(ManagedFeedImage)
+final class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: ManagedCache
```

Comment:
`cache` property is marked as optional in `FeedStore.xcdatamodel`, please make it optional to make the model consistent.

```suggestion
	@NSManaged var cache: ManagedCache?
```